### ISSUE TITLE
Some XML surveys contains some odd Javascript

### ISSUE THREAD

**Issue by ghachey** [PacificEMIS/pacific-emis-education-survey-tool#18]
See below bug. This should never happen. But we observe some surveys with such added script element into the survey. We need to understand why and fix.

![Screenshot 2021-06-09 090650](https://user-images.githubusercontent.com/1091769/121277526-5ef2fe00-c903-11eb-9e2b-614abd26efbb.jpg)

**Comment by ghachey** [comment]
From Omega-R:

> Where do you see this issue - in the admin panel or download from the mobile app DEBUG: show storage section? We were not able to reproduce the issue with our test accounts.

**Comment by ghachey** [comment]
I can no longer find which one. Lower priority of this. It could be no longer a problem in more recent version. I will observe and reporting here.

**Comment by maratomega** [comment]
Hi @ghachey did you have a chance to observe this issue?

**Comment by ghachey** [comment]
@maratomega no, lower priority and not affecting anything if it still even exist. Leave it for now.

**Comment by ghachey** [comment]
No longer seeing this and does not seem to affect anything. Closing.
